admin/RELEASE.md

```diff
@@ -53,12 +54,16 @@ git clone git@github.com:codeigniter4/userguide.git
     * Description: blank
 * Merge the PR then create a new Release:
     * Version: "v4.x.x"
+    * Target: master
     * Title: "CodeIgniter 4.x.x"
     * Description:
 ```
 CodeIgniter 4.x.x release.
 
 See the changelog: https://github.com/codeigniter4/CodeIgniter4/blob/develop/CHANGELOG.md
+
+## New Contributors
```

Comment:
Thanks for adding this! I've been leaving it when I clear out the rest, I think it's a nice touch to have some "shoutouts".